cf/read_write/netcdf/netcdfread.py

```diff
@@ -503,3 +510,69 @@ def _create_cfanetcdfarray(
         array = self.implementation.initialise_CFANetCDFArray(**kwargs)
 
         return array, kwargs
+
+    def _parse_chunks(self, ncvar):
+        # Parse dask chunks
+        """Parse the dask chunks.
+
+        .. versionadded:: TODODASKVER
+
+        :Parameters:
+
+            ncvar: `str`
+                The name of the netCDF variable containing the array.
+
+        :Returns:
+
+            `str`, `int` or `dict`
+                The parsed chunks that are suitable for passing to a
+                `Data` object containing the variable's array.
+
+        """
+        g = self.read_vars
+
+        default_chunks = "auto"
+        chunks = g.get("chunks", default_chunks)
+
+        if chunks is None:
+            return -1
+
+        if isinstance(chunks, dict):
+            if not chunks:
+                return default_chunks
+
+            # For ncdimensions = ('time', 'lat'):
+            #
+            # {} -> ["auto", "auto"]
+            # {'ncdim%time': 12} -> [12, "auto"]
+            # {'ncdim%time': 12, 'ncdim%lat': 10000} -> [12, 10000]
+            # {'ncdim%time': 12, 'ncdim%lat': "20MB"} -> [12, "20MB"]
+            # {'ncdim%time': 12, 'latitude': -1} -> [12, -1]
+            # {'ncdim%time': 12, 'Y': None} -> [12, None]
+            # {'ncdim%time': 12, 'ncdim%lat': (30, 90)} -> [12, (30, 90)]
+            # {'ncdim%time': 12, 'ncdim%lat': None, 'X': 5} -> [12, None]
```

Comment:
What is this comment block referring to, specifically? And can/should it perhaps ideally be subsumed into the docstring, as concrete examples?

Reply:
(For v3.14 feel free to ignore this comment, though, it's a bit fastidious.)

Reply:
Good point. It's not easily translatable to the docstring, and it is an underscore method ... but I have added a `chunks=` to each line to make it a bit clearer (https://github.com/NCAS-CMS/cf-python/pull/522/commits/22c6a2de02216cc6fae5cdf6605ae7be75c8516d). Happy to revisit after 3.14.0.